### DISCUSSION TITLE
steam-fix-raskladki

### DISCUSSION THREAD

**Discussion by giscus** [AMD-NICK/blog.amd-nick.me#29] [Blog comments]
# steam-fix-raskladki

После запуска игры в Steam у меня автоматически добавляется лишняя раскладка клавиатуры. Чтобы удалить ее из "карусели" Alt-Shift, приходится заходить в настройки, самому добавлять ее, затем сразу же удалять. И так после каждого рестарта компьютера

https://blog.amd-nick.me/steam-fix-raskladki

**Comment by S1ROZHA** [comment]
Сегодня после накатки свежей 11 стим добавил африканский)))))))))
Фикс: добавил его и удалил)
